modulefiles/clas12root/1.8.5

```diff
@@ -1,7 +1,7 @@
 #%Module1.0
 
 prereq hipo@4.1.0,4.2.0
-prereq qadb@3.0.0
+prereq qadb@3.0.0,3.1.0
```

Comment:
@baltzell can we instead specify 'qadb >= 3.0.0' or 'qadb@v3.x' or something similar, so we don't have to maintain a list here? Ditto for `hipo`.

Reply:
last i checked, only lists are supported:(